CONFIG

```diff
@@ -68,3 +68,4 @@ CONFIG_RDMA?=n
 # Build Ceph RBD support in bdev modules
 # Requires librbd development libraries
 CONFIG_RBD?=n
+CONFIG_NVME_IO_TIMEOUT=30
```

Comment:
Can this be a per-controller that is configured at runtime via a new field in spdk_nvme_ctrlr_opts?

I don't think it makes sense as a compile-time constant in CONFIG.